test/e2e/storage/vsphere/vsphere_scale.go

```diff
@@ -90,6 +89,7 @@ var _ = utils.SIGDescribe("vcp at scale [Feature:vsphere] ", func() {
 		if len(nodes.Items) < 2 {
 			framework.Skipf("Requires at least %d nodes (not %d)", 2, len(nodes.Items))
 		}
+		//nodeInfo = TestContext.NodeMapper.GetNodeInfo(nodes.Items[0].Name)
```

Comment:
can this code be deleted now?

Reply:
No this can not be removed. We need nodes and not nodeInfo objects here. Later in the code we need to pass nodes to `createNodeLabels` function.